datafusion/core/src/physical_plan/windows/window_agg_exec.rs

```diff
@@ -122,7 +122,9 @@ impl ExecutionPlan for WindowAggExec {
     }
 
     fn output_ordering(&self) -> Option<&[PhysicalSortExpr]> {
-        self.input.output_ordering()
+        // This executor maintains input order, and has required input_ordering filled
+        // hence output_ordering would be `required_input_ordering`
+        self.required_input_ordering()[0]
```

Comment:
I suggest changing this to `self.input.output_ordering()` to make the intent clearer

I realize that `required_input_ordering()` should produce the same result